packages/react-devtools-extensions/src/__tests__/updateMockSourceMaps.js

```diff
@@ -9,16 +9,23 @@ const {
 } = require('fs');
 const {emptyDirSync} = require('fs-extra');
 const {resolve} = require('path');
+const rollup = require('rollup');
+const babel = require('rollup-plugin-babel');
+const commonjs = require('rollup-plugin-commonjs');
+const jsx = require('acorn-jsx');
+const rollupResolve = require('rollup-plugin-node-resolve');
```

Comment:
Probably worth adding these new dependencies as `devDependencies` in `packages/react-devtools-extensions/package.json`